samples/iree-reduce/interesting.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+OUT1=`./tools/iree-compile $1 --iree-hal-target-backends=cuda --iree-opt-const-expr-hoisting=false --iree-opt-const-eval=false --compile-to=executable-targets | grep "llvm.intr.vector.reduce.fmax"`
```

Comment:
you'll want to use vmvx as a target for things as CUDA is not always available - we try to keep most samples/tests (especially for tools like this that are generally useful) working in all configurations